content/800-data-platform/200-projects/110-create-new-application-from-template.mdx

```diff
@@ -0,0 +1,105 @@
+---
+title: 'Create a new project from a template'
+metaTitle: 'Create a new Node.js application project from a template in the Prisma Data Platform'
+metaDescription: 'If you want to explore how to create projects in the Prisma Data Platform, you can start with template Prisma schema which is immediately migrated to an empty database.'
+tocDepth: 3
+---
+
+<TopBlock>
+
+If you want to explore how to create projects in the Prisma Data Platform, you can start with template Prisma schema which is immediately migrated to an empty database.
```

Comment:
I am not sure here whether "template Prisma schema" is singular or plural. Should this be "**a** template Prisma schema" or (if it's plural) "which **are** immediately migrated..."